### PR TITLE
Neferian head gob spawntime updated to 2 hours

### LINKED ISSUE #1702: Hanging Head of Nefarian in Stormwind Disappears after 1 Min
##### SMALL DESCRIPTION:

The hanging head of Nefarian in Stormwind disappears after 1 min of player handing in the quest and getting the buff.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

The head should display for 2 hours as agreed in https://github.com/azerothcore/azerothcore-wotlk/pull/1662.

##### CURRENT BEHAVIOUR:

The head only displays for about 60 seconds and then disappears.

##### STEPS TO REPRODUCE THE PROBLEM:

1. .use an Alliance character and then .additem 19003
2.  start the quest from the item
3.  follow the quest through
4.  by the end you will see the head disappears after roughly 60 seconds of hanging.

##### EXTRA NOTES:

N/A
##### BRANCH(ES):

master

##### AC HASH/COMMIT:

97e7e22

##### OPERATING SYSTEM:

Win 7/10/Svr 2016

##### MODULES:

N/A

##### OTHER CUSTOMIZATIONS:

N/A

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/72589833-hanging-head-of-nefarian-in-stormwind-disappears-after-1-min?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Falstring**
Neferian Head gob in SW updated spawn time

##### CHANGES PROPOSED:

spawntime from nef head gob in sw updated to 2 hours.

###### ISSUES ADDRESSED:
despawns after 21 seconds on current

Closes 
#1702

##### TESTS PERFORMED:
no errors/warnings produced in db.

##### HOW TO TEST THE CHANGES:
.use an Alliance character and then .additem 19003
start the quest from the item
follow the quest through
head should remain for 2 hours.

##### Target branch(es):

Master

### CODE REVIEW

**Comment by poszer** [comment]
You should update just one line...

**Comment by BarbzYHOOL** [comment]
Thank you for the PR.

**Comment by poszer** [comment]
Change your 7200 ( 7200 ms = 0.002 h )  to 7200000 (7200000 ms = 2 h)

https://github.com/azerothcore/azerothcore-wotlk/blob/49e8979e0c3a6ba79436256e7e3dc192385a15e2/data/sql/updates/pending_db_world/rev_1554920580112420800.sql#L18

File: data/sql/updates/pending_db_world/rev_1554920580112420800.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+ -- Actionlist SAI
+SET @ENTRY := 1472100;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=9;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@ENTRY,9,0,0,0,0,100,0,1000,1000,0,0,83,2,0,0,0,0,0,1,0,0,0,0,0,0,0,'Field Marshal Afrasiabi - On Script - Remove Npc Flag Questgiver'),
+(@ENTRY,9,1,0,0,0,100,0,5000,5000,0,0,1,0,8000,0,0,0,0,12,1,0,0,0,0,0,0,'Field Marshal Afrasiabi - On Script - Say Line 0'),
+(@ENTRY,9,2,0,0,0,100,0,8000,8000,0,0,1,1,10000,0,0,0,0,12,1,0,0,0,0,0,0,'Field Marshal Afrasiabi - On Script - Say Line 1'),
+(@ENTRY,9,3,0,0,0,100,0,3000,3000,0,0,50,179882,7200,1,0,0,0,8,0,0,0,-8925.57,496.042,103.767,2.42801,'Field Marshal Afrasiabi - On Script - Summon Gameobject The Severed Head of Nefarian and despawn after 2 hours'),
```
Comment (poszer) [inline data/sql/updates/pending_db_world/rev_1554920580112420800.sql]:
```suggestion
(@ENTRY,9,3,0,0,0,100,0,3000,3000,0,0,50,179882,7200000,1,0,0,0,8,0,0,0,-8925.57,496.042,103.767,2.42801,'Field Marshal Afrasiabi - On Script - Summon Gameobject The Severed Head of Nefarian and despawn after 2 hours'),
```

Comment (Nefertumm) [reply data/sql/updates/pending_db_world/rev_1554920580112420800.sql]: Just make an update query for this, please :)

Comment (Falstring) [reply data/sql/updates/pending_db_world/rev_1554920580112420800.sql]: > Change your 7200 ( 7200 ms = 0.002 h ) to 7200000 (7200000 ms = 2 h)
> 
> [azerothcore-wotlk/data/sql/updates/pending_db_world/rev_1554920580112420800.sql](https://github.com/azerothcore/azerothcore-wotlk/blob/49e8979e0c3a6ba79436256e7e3dc192385a15e2/data/sql/updates/pending_db_world/rev_1554920580112420800.sql#L18)
> 
> Line 18 in [49e8979](/azerothcore/azerothcore-wotlk/commit/49e8979e0c3a6ba79436256e7e3dc192385a15e2)
> 
>  (@ENTRY,9,3,0,0,0,100,0,3000,3000,0,0,50,179882,7200,1,0,0,0,8,0,0,0,-8925.57,496.042,103.767,2.42801,'Field Marshal Afrasiabi - On Script - Summon Gameobject The Severed Head of Nefarian and despawn after 2 hours'),

This is wrong. It's measured in seconds. Feel free to attempt on a test server yourself.

**Comment by Falstring** [comment]
> Change your 7200 ( 7200 ms = 0.002 h ) to 7200000 (7200000 ms = 2 h)
> 
> [azerothcore-wotlk/data/sql/updates/pending_db_world/rev_1554920580112420800.sql](https://github.com/azerothcore/azerothcore-wotlk/blob/49e8979e0c3a6ba79436256e7e3dc192385a15e2/data/sql/updates/pending_db_world/rev_1554920580112420800.sql#L18)
> 
> Line 18 in [49e8979](/azerothcore/azerothcore-wotlk/commit/49e8979e0c3a6ba79436256e7e3dc192385a15e2)
> 
>  (@ENTRY,9,3,0,0,0,100,0,3000,3000,0,0,50,179882,7200,1,0,0,0,8,0,0,0,-8925.57,496.042,103.767,2.42801,'Field Marshal Afrasiabi - On Script - Summon Gameobject The Severed Head of Nefarian and despawn after 2 hours'),

This is wrong. It is measured in seconds. Feel free to test yourself.

**Comment by comix1988** [comment]
I also completed the PR test just 2 minutes ago and I confirm the deaspawn after 2 hours.

good job @Falstring  thanks for test@gtao725

**Comment by gtao725** [comment]
@comix1988 no probs, although I still had to change line 18 from 7200 to 7200000. Pls do not forget to edit that before the merge, thx!

**Comment by poszer** [comment]
Actually it's used same code from my PR : 
https://github.com/azerothcore/azerothcore-wotlk/commit/6d0e54296f60082fc6a034203e4c3884ce70bcdf
Instead of C/P everything and runing same queries, just delete last line and update this : 
https://github.com/azerothcore/azerothcore-wotlk/blob/6d0e54296f60082fc6a034203e4c3884ce70bcdf/data/sql/updates/pending_db_world/rev_1554113581920680700.sql#L18

to this : 

https://github.com/azerothcore/azerothcore-wotlk/blob/49e8979e0c3a6ba79436256e7e3dc192385a15e2/data/sql/updates/pending_db_world/rev_1554920580112420800.sql#L18

**Comment by poszer** [comment]
It's tested in game also, everything is fine now. The gob is despawned after 2 hours.

**Pull request merged**